src/PKSim.Core/Services/SimulationParametersUpdater.cs

```diff
@@ -26,10 +27,10 @@ public SimulationParametersUpdater(IParameterSetUpdater parameterSetUpdater, IEn
          _entityPathResolver = entityPathResolver;
       }
 
-      public ValidationResult ReconciliateSimulationParametersBetween(Simulation sourceSimulation, Simulation targetSimulation)
+      public ValidationResult ReconciliateSimulationParametersBetween(Simulation sourceSimulation, Simulation targetSimulation, PKSimBuildingBlockType buildingBlockType = PKSimBuildingBlockType.Simulation)
```

Comment:
This is really the only changes
We typically use this methods to copy ALL Simul;ation parameters from one simulation to another (after clone for example, where all buidlign blocks were in teh green state)
in this case however, we are using the same method but the BB do not have to be green, which leads to protocol being outdated

Protocol is actually the only bb that creates issue because of its 1 to n relation ship. All others are working fine.